game_progress.py

```diff
@@ -0,0 +1,75 @@
+import random
+from choosing_sides import Signs
+
+
+def first_turn_move(
+    chosen_side: str,
+    signs: Signs,
+    num_of_lines: int,
+    field: list[list[Signs | None]],
+    player_sign: Signs,
+) -> None:
+    for line in range(num_of_lines):
+        if field[line] == [None] * num_of_lines and chosen_side == signs.crosses.name:
+            return ask_player_for_coordinates(player_sign, field)
+
+
+def ask_player_for_coordinates(
+    player_sign: Signs, field: list[list[Signs | None]], sign_not_placed: bool = True
+) -> None:
+    print("---------Your turn!---------")
+    while sign_not_placed:
+        try:
+            line_coordinate, column_coordinate = input(
+                "Enter coordinates with space: "
+            ).split()
+            line_coordinate, column_coordinate = int(line_coordinate), int(
+                column_coordinate
+            )
+            if (
+                check_for_sign_in_coordinate(field, line_coordinate, column_coordinate)
+                is False
```

Comment:
`if not check_for_sign_in_coordinate(field, line_coordinate, column_coordinate)`